docs/docsite/README.md

```diff
@@ -14,6 +14,12 @@ such as link references, you may install sphinx and build the documentation by r
 To include module documentation you'll need to run `make webdocs` at the top level of the repository.  The generated
 html files are in docsite/htmlout/.
 
+To limit module documentation building to a specific module, run `MODULES=NAME
+make webdocs` instead. This should make testing module documentation syntax much
+faster. Instead of a single module, you can also specify a comma-separated list
+of modules. In order to skip building documentation for all modules, specify
+non-existing module name, for example `MODULES=none make webdocs`.
```

Comment:
I recommend passing vars as args in all new `make` examples. Example: `make webdocs MODULES=none`

Also, use `a non-existent module name` instead of `non-existing module name`.

Not a blocker, just a suggestion to improve readability. If you do change it, update all new occurrences.